README.md

```diff
@@ -1,93 +1,83 @@
 # [![Moon](https://raw.githubusercontent.com/polyfox/moon/master/moon-logo.png)](https://raw.githubusercontent.com/polyfox/moon/master/moon-logo.png) Moon
 [![Build Status](https://travis-ci.org/polyfox/moon.svg?branch=master)](https://travis-ci.org/polyfox/moon)
+[![Code Climate](https://codeclimate.com/github/polyfox/moon/badges/gpa.svg)](https://codeclimate.com/github/polyfox/moon)
 
 ## Introduction
-Moon is a experimental prototyping 2D Game Engine written in C++ and scripted using
-mruby.
+Moon is a experimental prototyping 2D Game Engine written in C++ and scripted using [mruby](https://github.com/mruby/mruby).
 
-## Projects using Moon
-Lunar Howl (MMO RPG)
-es-moon (Roguelike RPG)
-Lunar Metal (2D Top-Down RTS)
 
-## Dependencies
-Moon ships with most of its dependencies in the vendor directory
-```
-freetype-gl
-glfw (3)
-glm
-gorilla-audio
-mruby
-soil
-```
+## Supported
+Currently testing is done linux, moon may not run or even compile on other systems. If you managed to get Moon running on your system, please open a PR and share your changes to get it built.
 
-## Building
-If you are downloading moon for the first time from git you can use:
-```
-git clone --recursive https://github.com/archSeer/moon.git
-```
-This will grab moon and all its submodules.
 
+## Requirements
+While moon attempts to vendor most of its dependencies, here is a list of 
+things you'll need before getting started.
 
 
+### Dependencies
+```
+# building
+premake4 
+cmake >= 2.8.9
+ruby >= 1.9.2
 
+# runtime
+OpenAL
+OpenGL
 
-__ORIGINAL_README__
-## Moon and Lunar Howl
+# For dependency docs
+doxygen
 
-Moon is the name of the engine (libmoon), running OpenGL, GLFW and mruby.
+# for mruby-yaml
+yaml
+```
 
-Lunar Howl is the name of the going-to-be-built game on top of the Moon engine.
 
-Requires GLFW3 externally installed. Manually compiles and statically links
-mruby, gorilla audio, GLM and soil.
+### Features required
+In order to use Moon, a OpenGL 2.1 or greater, capable system is required,
+we will make no effort to support anything less than 2.1.
 
-NOTE: To define mruby C extensions, you need to include a `void moon_init_mrb_ext(mrb_state *mrb)`
-function, that does the same form of initialization as an "Init_<name>" function would do. This
-function needs to be declared (and just left empty) even if you don't use any mruby C code, or
-the compiler will crash.
 
-## Support
+## Building
 
-We support graphics cards with OpenGL 2.1 support and the following extensions:
```

Comment:
Add this back in or note it somewhere else, because this information is also useful to us as developers (I keep forgetting which extensions we use)